csharp/src/Microsoft.ML.OnnxRuntime/InferenceSession.cs

```diff
@@ -529,6 +529,17 @@ public string EndProfiling()
             }
         }
 
+        /// <summary>
+        /// Return the nanoseconds of profiling's start time
+        /// 
```

Comment:
/// </summary>

Also maybe consider changing this to a property, instead of a method?

Reply:
Good point. Updated